apps/scheduler/lib/job/job.ex

```diff
@@ -0,0 +1,79 @@
+defmodule Scheduler.Job do
+  @moduledoc """
+  Module defining job functions
+  """
+  use Database
+  require Amnesia.Helper
+
+  @doc """
+  Creates a new Job. It takes a name, a time to execute (in erlang system
+  time, seconds) and a tuple of {module, fun, args} that will be executed
+  when the job is run.
+  """
+  def create(name, execute_at, mfa) do
+    Amnesia.transaction do
+      %Job{
+        name: name,
+        execute_at: execute_at,
+        params: :erlang.term_to_binary(mfa),
+        status: "pending",
```

Comment:
great idea. yea we will need to decide how to handle these

Reply:
A quick but still reasonable option could be to let the client determine when to delete them, providing an API to do so and leaving it at that